CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+# FIXME: Work out which CMake version we actually need.
 cmake_minimum_required(VERSION 3.5)
 
 # FIXME: This silences a warning but I'm not sure why its needed.
```

Comment:
https://stackoverflow.com/a/32746363

The default changed at CMake 3.0, so this should be explicitly set. Which we do.

I think this comment can go away / be replaced by one that explains what we're doing.

Reply:
Comment removed. Thanks for investigating this.